dds_cli/utils.py

```diff
@@ -330,3 +338,25 @@ def delete_folder(folder):
         else:
             file_or_folder.unlink()
     folder.rmdir()
+
+
+def __project_creation_error(response_json: Dict) -> str:
+    message, title, description, pi, email = (
+        response_json.get("message"),
+        response_json.get("title"),
+        response_json.get("description"),
+        response_json.get("pi"),
+        response_json.get("email"),
+    )
+
+    messages: List = [message, title, description, pi, email]
+
+    error = next(message for message in messages if message)
+
+    if isinstance(error, List):
+        return error[0]
+
+    if "Insufficient credentials" in error:
+        return "You do not have the required permissions to create a project."
```

Comment:
The code never reaches this part 
```
Current user: researchuser_1
ERROR    ⚠️ Failed to create project: Insufficient credentials ⚠️        
```
The "insufficient credentials" part is printed out from line 204.

I feel like this could be a separate issue though. I think you have solved the other error printouts in a good way so if you want to remove this part we can make a card to go through "Insufficient Credentials" errors in general within the entire CLI at a later time.